Added foreach so we can retrieve $serverID from multiple input

xerod opened the pull request:
Solved my #1  problem when using multiple input or `allowMultiple` options and only retrieve one $serverID when using $request->get('file'). Just to make sure, you have to re-configure you `Filepond.setOptions `to: 

```
FilePond.setOptions({
    name: 'file[]',
    server: window.location.protocol+'//'+window.location.hostname+'/filepond/api/process',
})
```

Sopamo commented on src/Http/Controllers/FilepondController.php:
```diff
@@ -29,15 +29,15 @@ public function __construct(Filepond $filepond)
+            if (!$file->move($filePathParts['dirname'], $filePathParts['basename'])) {
+                return Response::make('Could not save file', 500);
+            }
+            return Response::make($this->filepond->getServerIdFromPath($filePath), 200);
```
This can't work, can it? The return inside of the foreach loop would end the function execution after moving the first image.

Sopamo commented:
Thank you for your pull request! Could you have a look at the code again?

xerod commented:
This works fine, my first assumption is: even it end the function, the second one still executed because filepond processed it one by one (if it's two images, filepond would process it two times). This should tell:

<img width="926" alt="image" src="https://user-images.githubusercontent.com/32943023/53753206-43f2e180-3ee3-11e9-8ce2-108d68efaf6c.png">

<img width="1260" alt="image" src="https://user-images.githubusercontent.com/32943023/53752763-0e012d80-3ee2-11e9-914d-f4a8b19ab122.png">

I added the foreach at FileondController.php so the upload function can process the array even it's returning at the first loop. If i removed it (using your original code), it's basically give an exception like this:

<img width="935" alt="image" src="https://user-images.githubusercontent.com/32943023/53752003-2d975680-3ee0-11e9-8da9-6d6d7154ebea.png">

Bottom line: the foreach didn't do multiple loop, it's basically extract the array that filepond give (one at a time).

As long you're using the array name `file[]` for the input (like what they did at filepond PHP boilerplate), you're good to go. I don't know if this is the most efficient way or not. It's still nice to have this package already, saved more time than it should. Thanks for replying tho

Sopamo commented:
Okay, that makes more sense now. I was wondering how filepond would handle the response for multiple files. If you like, you could refactor your code so it just gets the first element instead of looping over the array (because we aren't actually looping, as you said). If you don't have the time, I can merge your PR and do it myself.

xerod commented:
do you mean like this?:

```
    public function upload(Request $request)
    {
        $file = $request->file('file')[0];

        $filePath = tempnam(config('filepond.temporary_files_path'), "laravel-filepond");
        $filePathParts = pathinfo($filePath);

        if(!$file->move($filePathParts['dirname'], $filePathParts['basename'])) {
            return Response::make('Could not save file', 500);
        }
        return Response::make($this->filepond->getServerIdFromPath($filePath), 200);
    }
```

looks more cleaner to me.

Sopamo commented:
Perfect, thanks!

xerod commented:
Don't forget to update your readme.md @Sopamo

bfuchs commented:
This change breaks functionality for me when uploading single files. I get the following error in /src/Http/Controllers/FilepondController.php:

> Cannot use object of type Illuminate\Http\UploadedFile as array

`$request->file('file')` returns an no array.
